### DISCUSSION TITLE
Scripts configuration

### DISCUSSION THREAD

**Discussion by mmh352** [mmh352/ou-container-builder#21] [General]
With the scripts section:

```
scripts:
  - inline:
    - wget --no-check-certificate https://github.com/OpenRefine/OpenRefine/releases/download/3.3/openrefine-linux-3.3.tar.gz
    - mkdir /var/openrefine
    - tar -xzf openrefine-linux-3.3.tar.gz --directory /var/openrefine
    - rm openrefine-linux-3.3.tar.gz
 ```

does this get sent to a single layer (I haven't checked in the generated Dockerfile)?

What's the rationale for one line per command? Clarity?

_Originally posted by @psychemedia in https://github.com/mmh352/ou-container-builder/issues/11#issuecomment-832073779_

**Comment by mmh352** [comment]
Yes, this gets sent as a single layer. Having one command per line is more readable, but it will also work if all commands are given in a single line separated by "&&".

**Comment by psychemedia** [comment]
Presumably the order is respected? (I'm pretty sure it is, just trying to air and raise any possible confusion, bad assumptions etc).

**Comment by mmh352** [comment]
Yes. The order is respected.
